R/rule_fit.R

```diff
@@ -12,7 +12,15 @@ xrf_fit <-
            gamma = 0,
            subsample = 1,
            lambda = 0.1,
+           counts = TRUE,
            ...) {
+    mtry <-
+      process_mtry(
+        colsample_bytree = colsample_bytree,
+        counts = counts,
+        n_predictors = length(attr(terms(formula), "term.labels")),
```

Comment:
I'm guessing there's a more tidymodels-esque way of doing this—what's the best way of counting the number of predictors when passed a formula and data? Elsewhere, where the inputs are `x` and `y`, we can just take `ncol(x)`.